src/ExpenseTrackerGrupo2/Api/Controllers/ExpenseController.cs

```diff
@@ -1,6 +1,4 @@
-using System;
-
-namespace ExpenseTrackerGrupo2.src.ExpenseTrackerGrupo2.Api.Controllers;
+namespace ExpenseTrackerGrupo2.Api.Controllers;
```

Comment:
Why do you change the names of the namespaces? That's how the folder structure is, but I don't understand why you delete it.

Reply:
Have you considered we have duplicate words in the namespace? Such as ExpenseTrackerGrupo2 at the begging and in the middle. So it doesn't make sense of it. Then it's not common, at least, in .NET project to see a namespace using src on it. It normally goes with the artifact id or project name and then the rest of tree structure.